docs/09-composer-integration.md

```diff
@@ -0,0 +1,13 @@
+# Composer Integration
+
+Media Atom Maker integrates with Composer via the API. A new Composer piece can be made via the "Create Video Page" button in the atom editor, this will use the [createContent](https://github.com/guardian/flexible-content/blob/main/flexible-content-apiv2/src/main/scala/com/gu/flexiblecontent/apiv2/dispatcher/ApiDispatcher.scala#L373) endpoint to generate a new video article with the [provided fields](https://github.com/guardian/flexible-content/blob/main/flexible-content-common/src/main/scala/com/gu/flexiblecontent/model/MAMVideoPageInformation.scala). ßSubsequent changes to the atom will be reflected in Composer via update requests made to the [videoPage](https://github.com/guardian/flexible-content/blob/main/flexible-content-apiv2/src/main/scala/com/gu/flexiblecontent/apiv2/dispatcher/ApiDispatcher.scala#L630) endpoint.
```

Comment:
```suggestion
Media Atom Maker integrates with Composer via the API exposed by the `apiv2` project. A new Composer piece can be made via the "Create Video Page" button in the atom editor, this will use the [createContent](https://github.com/guardian/flexible-content/blob/main/flexible-content-apiv2/src/main/scala/com/gu/flexiblecontent/apiv2/dispatcher/ApiDispatcher.scala#L373) endpoint to generate a new video article with the [provided fields](https://github.com/guardian/flexible-content/blob/main/flexible-content-common/src/main/scala/com/gu/flexiblecontent/model/MAMVideoPageInformation.scala). ßSubsequent changes to the atom will be reflected in Composer via update requests made to the [videoPage](https://github.com/guardian/flexible-content/blob/main/flexible-content-apiv2/src/main/scala/com/gu/flexiblecontent/apiv2/dispatcher/ApiDispatcher.scala#L630) endpoint.
```

Reply:
oh, yes, that makes sense – I was trying to remember which one did auth, that'll be `composer-backend`. Good to get it documented regardless – and we may want to change those links to endpoint code as a result.